src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -293,6 +293,11 @@ public override async ValueTask<IReadOnlyList<BulkStateItem>> GetBulkStateAsync(
                 Parallelism = parallelism ?? default(int)
             };
 
+            if (metadata != null)
+            {
+                getBulkStateEnvelope.Metadata.Add(metadata);
```

Comment:
add UT

Reply:
A ValidateRequest test is added for the bulk operation.